src/app/AppContainer.js

```diff
@@ -87,6 +87,7 @@ class AppContainer extends React.Component<PropsType> {
 		return (
 			<MainApp
 				rootClientId={ this.props.rootClientId }
+				editTitle={ this.props.editTitle }
 				serializeToNativeAction={ this.serializeToNativeAction }
 				toggleHtmlModeAction={ this.toggleHtmlModeAction }
 				setTitleAction={ this.setTitleAction }
```

Comment:
do we need to pass both setTitleAction and editTitle? it looks to me like setTitleAction can be eliminated. Also the function itself?
```
	setTitleAction = ( title: string ) => {
		this.props.editTitle( title );
	};
```

Reply:
Removed `editTitle` which wasn't meant to be there.  This PR got a bit confusing since I couldn't really completely remove `setTitleAction`.